partner_medical_medication/views/res_partner_view.xml

```diff
@@ -12,23 +12,17 @@
         <field name="inherit_id" ref="base.view_partner_form"/>
         <field name="arch" type="xml">
             <xpath expr="//div[@name='button_box']" position="inside">
-                <field name="patient_ids" invisible="True"/>
-                <button  
-                    name="partner_medical_medication.partner_action_medication" type="action" 
-                    class="oe_stat_button" icon="fa-list"
-                    attrs="{'invisible': [('patient_ids', '=', [])]}">
-                    <span>Medications</span>
-                </button>
+                <field name="medication_ids" invisible="True" />
+                <button name="medical_medication.medical_patient_medication_action"
+                        string="Medications"
+                        type="action"
+                        class="oe_stat_button"
+                        icon="fa-list"
+                        context="{'search_default_id': medication_ids}"
+                        attrs="{'invisible': [('medication_ids', '=', [])]}"
```

Comment:
That's kind of what I was thinking - physicians won't really have any of this, and we might want to instead add some sort of contextual something's for them instead (appointments maybe?)

What I really tried to do was create a `Medical` button instead, which would direct to a view showing all the medical entities for the partner. One would assume a partner only has entities of a certain type (patients probably aren't physicians), so basically it would allow for a more versatile button.

This plan failed miserably though due to the need to link to models of disparate types in the same view, even if the attributes are the same. Really made me made.